telegram/request.py

```diff
@@ -382,17 +383,18 @@ def retrieve(self, url: str, timeout: float = None) -> bytes:
 
         return self._request_wrapper('GET', url, **urlopen_kwargs)
 
-    def download(self, url: str, filename: str, timeout: float = None) -> None:
+    def download(self, url: str, filepath: Union[Path, str], timeout: float = None) -> None:
         """Download a file by its URL.
 
         Args:
             url (:obj:`str`): The web location we want to retrieve.
+            filepath (:obj:`Path` | :obj: `str`): The filepath to download the file to.
```

Comment:
```suggestion
            filepath (:obj:`pathlib.Path` | :obj: `str`): The filepath to download the file to.
```
doing this will help resolve docs rendering when we enable intersphinx.

Reply:
done